assets/scss/_styles_project.scss

```diff
@@ -4,8 +4,8 @@
 @import "registry";
 @import "tabs";
 
-.navbar-position-absolute {
-  position: absolute !important;
+.td-navbar {
+    background: $primary !important;
```

Comment:
IMHO, it never really worked to have a transparent top nav given the current homepage design. By default Docsy will set the top-nav background to the `$primary` color but only after the user has scrolled past the `blocks/cover`. This change sets the background to always be the primary color.